src/Control/Monad/Aff.purs

```diff
@@ -73,15 +73,15 @@ cancelWith aff c = runFn3 _cancelWith nonCanceler aff c
 -- | If you do need to handle exceptions, you can use `runAff` instead, or
 -- | you can handle the exception within the Aff computation, using
 -- | `catchError` (or any of the other mechanisms).
-launchAff :: forall e a. Aff e a -> Eff (err :: EXCEPTION | e) Unit
-launchAff = runAff throwException (const (pure unit)) <<< liftEx
-  where
-  liftEx :: Aff e a -> Aff (err :: EXCEPTION | e) a
-  liftEx = _unsafeInterleaveAff
+launchAff :: forall e a. Aff (err :: EXCEPTION | e) a -> Eff (err :: EXCEPTION | e) (Canceler (err :: EXCEPTION | e))
```

Comment:
These changes look great. This is the only one I'm concerned about. Technically, an `EXCEPTION` effect type in an `Aff` doesn't mean anything, since `Aff` doesn't throw exceptions in a way that can be caught via the same mechanism for `Eff`.

The intention is that you can lift any `Eff` with `EXCEPTION` into an `Aff`, and it drops the `EXCEPTION`; and that you can lower any `Aff` into an `Eff` and it adds `EXCEPTION`. That's not quite precise but I think it's probably the best we can do.

Is there some other way of addressing your need for this change?